src/index.ts

```diff
@@ -10,36 +10,37 @@ const getFullURL = (requestPayload: any) => {
   return BASE_URL + params
 }
 
-const sendGA3Event = function (
-  eventType: string,
-  event: MCEvent,
-  settings: ComponentSettings
-) {
-  const requestPayload = getToolRequest(eventType, event, settings)
-
-  let ecommerceParams = {}
-  if (eventType === 'ecommerce') {
-    ecommerceParams = getEcommerceParams(event)
-  }
-
-  const finalURL = getFullURL({ ...requestPayload, ...ecommerceParams })
-  fetch(finalURL)
-
-  if (settings['ga-audiences'] || settings['ga-doubleclick']) {
-    gaDoubleClick(event, settings, finalURL)
+export default async function (manager: Manager, settings: ComponentSettings) {
+  const sendGA3Event = function (
+    eventType: string,
+    manager: Manager,
```

Comment:
you don't really need to receive the `manager` here, as it's available in the default function, on line 13, no?